Makefile

```diff
@@ -1,5 +1,8 @@
 SHELL := /bin/bash
 
+MATRIX_OS   ?= darwin linux
+MATRIX_ARCH ?= amd64
```

Comment:
IIRC This should not be necessary because a release build does not target Darwin, only Linux (handled by docker deps)